src/application/extension-popup-menu/popup-home-view/app.tsx

```diff
@@ -0,0 +1,75 @@
+import { useNavigate } from 'react-router-dom';
+
+import {
+  AGORA_LOGO,
+  POLYMARKET_LOGO,
+  SNAPSHOT_LOGO,
+  TALLY_LOGO,
+} from 'shared/web3';
+import { IDRISS_ICON } from 'shared/idriss';
+
+import { TRADING_COPILOT_LOGO } from '../constants';
+import { EXTENSION_POPUP_MENU_ROUTES } from '../app';
+
+import { SectionTile } from './components';
+
+export const App = () => {
+  const navigate = useNavigate();
+  return (
+    <div className="flex max-w-[570px] grow flex-col items-center bg-gray-100 px-1 pt-6">
```

Comment:
this 570px is already applied in parent, probably you don't need it here

Reply:
nice catch!